pkgs/by-name/ha/harmonia/package.nix

```diff
@@ -11,27 +13,29 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "harmonia";
-  version = "0.8.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = pname;
     rev = "refs/tags/${pname}-v${version}";
```

Comment:
```suggestion
    repo = "harmonia";
    rev = "refs/tags/harmonia-v${version}";
```
#277994

Reply:
Does this belong in the same update commit, or a separate one?